package.json

```diff
@@ -40,7 +40,7 @@
     "ember-cli-babel": "^7.22.1",
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-string-utils": "^1.1.0",
-    "ember-cli-typescript": "^3.1.3",
+    "ember-cli-typescript": "^4.0.0-rc.1",
```

Comment:
I believe 4.0.0 final is out, can you update?

Reply:
Will do!

Reply:
Updated, and for TS 4.0 as well.